platform/pwl.c

```diff
@@ -1103,6 +1115,93 @@ zxdg_shell_v6_ping (void                 *data G_GNUC_UNUSED,
 #endif /* HAVE_XDG_SHELL_UNSTABLE_V6 */
 
 
+static PwlSeat*
+pwl_seat_create(struct wl_seat *seat, uint32_t name, uint32_t version, PwlDisplay *display)
+{
+    PwlSeat *self = g_slice_new0 (PwlSeat);
+    self->seat = seat;
+    self->seat_name = name;
+    self->seat_version = version;
+
```

Comment:
This empty line is not needed, please remove it.

Reply:
removed